### PR TITLE
Add component user data

### PR DESCRIPTION
**Pull request by DmytroTikhostup**

### CODE REVIEW

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]
you took wrong branch for PR. Change branch from main to develop and resolve all conflicts. Versions all your files deprecated

**Comment by ArtemLigerko** [comment]
... in addition, make pull and merge develop branch into your branch before push it to Github

**Review by vasyagladush — APPROVED** [review]

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]
ho about to remove all comments?

File: src/components/FormSettingsUserData.tsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      setFileImage(e.currentTarget.files[0]);
+    }
+  };
+  //-----------------------------------------------------------------------
+
+  return (
+    <>
+      {/*----modal window -------*/}
+      <Modal show={show} onHide={handleClose}>
+        <Modal.Header closeButton>
```
Comment (Mykyta-Pysarev) [inline src/components/FormSettingsUserData.tsx:101]: You could use ModalUniversal to save some time.

File: src/components/FormSettingsUserData.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            <Form.Group className="mb-3" controlId="inputBirthDateUser">
+              <Form.Control
+                type="date"
+                placeholder="Date"
+                data-date-format="YYYY/MM/DD"
+              />
+            </Form.Group>
+            {/*----advanced settings button -------*/}
+            <div className="d-grid gap-2">
+              <Button variant="light">Switch to professional</Button>
```
Comment (Mykyta-Pysarev) [inline src/components/FormSettingsUserData.tsx]: Use n18i for the text. So your module could be swithed to another language.

File: src/components/FormSettingsUserData.tsx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+                    size="sm"
+                    ref={fileRefImage}
+                    onChange={handleImageUpload}
+                  />
+                </div>
+              </Form.Group>
+            </Figure>
+
+            {/*----NameUser input -------*/}
+            <Form.Group className="mb-3" controlId="inputNameUser">
```
Comment (Mykyta-Pysarev) [inline src/components/FormSettingsUserData.tsx:160]: Use n18i for the text in placeholders

**Review by Mykyta-Pysarev — CHANGES_REQUESTED** [review]
Please, use n18i to translate your module.

File: src/components/FormSettingsUserData.tsx (hunk 3 above)
Comment (DmytroTikhostup) [reply src/components/FormSettingsUserData.tsx:160]: done

File: src/components/FormSettingsUserData.tsx (hunk 2 above)
Comment (DmytroTikhostup) [reply src/components/FormSettingsUserData.tsx]: done

**Review by Mykyta-Pysarev — APPROVED** [review]

File: src/components/FormSettingsUserData.tsx (hunk 1 above)
Comment (DmytroTikhostup) [reply src/components/FormSettingsUserData.tsx:101]: is this a prerequisite?

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**